store/report.go

```diff
@@ -52,21 +36,59 @@ func (m *mongoDB) GetNearbyReportingUserCount(reportType schema.ReportType, dist
 			},
 		},
 	}
-	cursor, err := c.Aggregate(ctx, pipeline)
-	if err != nil {
-		return 0, err
-	}
+}
 
-	if !cursor.Next(ctx) {
-		return 0, nil
+// GetNearbyReportingUserCount returns the number of users who have reported symptoms/behaviors
+// in the specified area and within the specified day.
```

Comment:
Change the comment accordingly since now it returns count for both today and yesterday.

Reply:
OK